api/HTMLInputElement.json

```
@@ -1993,7 +1993,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false,
+              "version_added": "18",
               "impl_url": "https://webkit.org/b/234348"
```

Now that it is implemented, let's replace the `impl_url` with `notes` similar to Chrome:

```suggestion
              "version_added": "18",
              "notes": "Before Safari 18, a `selectionchange` event was fired on `Document`, see [`Document`'s `selectionchange` event](https://developer.mozilla.org/docs/Web/API/Document/selectionchange_event). See [bug 271033](https://webkit.org/b/271033) for firing the event on `<input>` elements."
```